impedance/tests/test_circuit_elements.py

```diff
@@ -46,20 +44,18 @@ def test_all():
             assert np.isclose(val, correct_vals[key]).all()
 
         # check for typing:
-        try:
+        with pytest.raises(AssertionError):
             f = circuit_elements['R']
             f(1, 2)
-        except(AssertionError):
-            pass
-        else:
-            raise Exception('unhandled error occurred')
+
         # test for handling more wrong inputs
-        try:
+        with pytest.raises(AssertionError):
             f(['hi'], ['yes', 'hello'])
-        except(AssertionError):
-            pass
-        else:
-            raise Exception('unhandled error occurred')
+
+    # Test no overflow in T at high frequencies
+    with pytest.warns(None) as record:
+        circuit_elements['T']([1, 2, 50, 100], [10000])
+    assert not record
```

Comment:
I get an error here running in a Jupyter Lab instance and running pytest locally. I get `AssertionError` from both approaches. Any idea why my machine would overflow while yours doesn't?

Reply:
I think the warnings within the context of `with pytest.warns(None)` are stored in record. By asserting `not record`, the check is to make sure that no warnings were raised

Reply:
Alright, then that's how I originally understood it. So I think that means my machine is giving overflow when yours isn't?